Client/mods/deathmatch/logic/rpc/CRPCFunctions.cpp

```diff
@@ -53,7 +53,7 @@ SFixedArray<CRPCFunctions::SElementRPCHandler, CRPCFunctions::NUM_RPC_FUNCS> CRP
 
 CRPCFunctions::CRPCFunctions(CClientGame* pClientGame)
 {
-    m_bShowRPCs = true;
+    m_bShowRPCs = false;

```

Comment:
i know this was an unintentional commit, but you can use the [`showrpcs`](https://github.com/multitheftauto/mtasa-blue/blob/e11685cab4beb7958ab202261f9c9d9b4ce71e58/Client/mods/deathmatch/CClient.cpp#L120) command for this.

Reply:
twas originally false, I just changed it for debugging.